Paystack/PublicHeaders/PSTCKCard.h

```diff
@@ -67,24 +67,24 @@ typedef NS_ENUM(NSInteger, PSTCKCardFundingType) {
 /**
  *  The Paystack ID for the card.
  */
-@property (nonatomic, readonly, nullable) NSString *cardId;
+@property (nonatomic, readonly, nullable) NSString *cardId __attribute__((deprecated("This SDK has been deprecated, Please refer to our new SDK: https://github.com/PaystackHQ/paystack-sdk-ios")));
```

Comment:
Could this repeated message be centralised so that any change to the messaging can be reflected everywhere?

Reply:
I could be but I would still need to do it per file, just ObjC things so I chose to copy paste it but this code base will not be worked on anymore after this so i think its okay.

Reply:
Yeah, was so minor, that I still approved anyway :)